exchanges/ftx/ftx_test.go

```diff
@@ -0,0 +1,1007 @@
+		t.Skip()
+	}
+	_, err := f.GetAccountInfo()
+	if err != nil {
+		t.Error(err)
+	}
+}
+
+func TestGetPositions(t *testing.T) {
+	t.Parallel()
+	if !areTestAPIKeysSet() {
+		t.Skip()
+	}
+	_, err := f.GetPositions()
+	if err != nil {
+		t.Error(err)
+	}
+}
+
+func TestChangeAccountLeverage(t *testing.T) {
+	t.Parallel()
+	if !areTestAPIKeysSet() || !canManipulateRealOrders {
+		t.Skip("skipping test, either api keys or manipulaterealorders isnt set correctly")
+	}
+	err := f.ChangeAccountLeverage(50)
+	if err != nil {
+		t.Error(err)
+	}
+}
+
+func TestGetCoins(t *testing.T) {
+	t.Parallel()
+	if !areTestAPIKeysSet() {
+		t.Skip()
+	}
+	_, err := f.GetCoins()
+	if err != nil {
+		t.Error(err)
+	}
+}
+
+func TestGetBalances(t *testing.T) {
+	t.Parallel()
+	if !areTestAPIKeysSet() {
+		t.Skip()
+	}
+	_, err := f.GetBalances()
+	if err != nil {
+		t.Error(err)
+	}
+}
+
+func TestGetAllWalletBalances(t *testing.T) {
+	t.Parallel()
+	if !areTestAPIKeysSet() {
+		t.Skip()
+	}
+	_, err := f.GetAllWalletBalances()
+	if err != nil {
+		t.Error(err)
+	}
+}
+
+func TestFetchDepositAddress(t *testing.T) {
+	t.Parallel()
+	if !areTestAPIKeysSet() {
+		t.Skip()
+	}
+	_, err := f.FetchDepositAddress("TUSD")
+	if err != nil {
+		t.Error(err)
+	}
+}
+
+func TestFetchDepositHistory(t *testing.T) {
+	t.Parallel()
+	if !areTestAPIKeysSet() {
+		t.Skip()
+	}
+	_, err := f.FetchDepositHistory()
+	if err != nil {
+		t.Error(err)
+	}
+}
+
+func TestFetchWithdrawalHistory(t *testing.T) {
+	t.Parallel()
+	if !areTestAPIKeysSet() {
+		t.Skip()
+	}
+	_, err := f.FetchWithdrawalHistory()
+	if err != nil {
+		t.Error(err)
+	}
+}
+
+func TestWithdraw(t *testing.T) {
+	t.Parallel()
+	if !areTestAPIKeysSet() || !canManipulateRealOrders {
+		t.Skip("skipping test, either api keys or manipulaterealorders isnt set correctly")
+	}
+	_, err := f.Withdraw("BTC", "38eyTMFHvo5UjPR91zwYYKuCtdF2uhtdxS", "", "", "957378", 0.01)
+	if err != nil {
+		t.Error(err)
+	}
+}
+
+func TestGetOpenOrders(t *testing.T) {
+	t.Parallel()
+	if !areTestAPIKeysSet() {
+		t.Skip()
+	}
+	_, err := f.GetOpenOrders(spotPair)
+	if err != nil {
+		t.Error(err)
+	}
+}
+
+func TestFetchOrderHistory(t *testing.T) {
+	t.Parallel()
+	if !areTestAPIKeysSet() {
+		t.Skip()
+	}
+	_, err := f.FetchOrderHistory(spotPair, time.Time{}, time.Time{}, "2")
+	if err != nil {
+		t.Error(err)
+	}
+	_, err = f.FetchOrderHistory(spotPair, time.Unix(1559881511, 0), time.Unix(1559901511, 0), "2")
+	if err != nil {
+		t.Error(err)
+	}
+	_, err = f.FetchOrderHistory(spotPair, time.Unix(1559901511, 0), time.Unix(1559881511, 0), "2")
+	if err == nil {
+		t.Error(err)
+	}
+}
+
+func TestGetOpenTriggerOrders(t *testing.T) {
+	t.Parallel()
+	if !areTestAPIKeysSet() {
+		t.Skip()
+	}
+	_, err := f.GetOpenTriggerOrders(spotPair, "")
+	if err != nil {
+		t.Error(err)
+	}
+}
+
+func TestGetTriggerOrderTriggers(t *testing.T) {
+	t.Parallel()
+	if !areTestAPIKeysSet() {
+		t.Skip()
+	}
+	_, err := f.GetTriggerOrderTriggers("alkdjfkajdsf")
```

Comment:
This 404's

Reply:
ah yeahhhhh oops the order ids need to be numbers. fixed now